samza-core/src/test/java/org/apache/samza/zk/TestZkBarrierForVersionUpgrade.java

```diff
@@ -23,11 +23,6 @@
 import org.apache.samza.config.ZkConfig;
 import org.apache.samza.testUtils.EmbeddedZookeeper;
 import org.apache.samza.util.NoOpMetricsRegistry;
-import org.junit.After;
```

Comment:
nit: since we are tentatively disabling this test, let's comment those import lines instead of delete them.

Reply:
Makes sense. Updated the patch.